api/src/test/resources/rdf/abox/filegraph/dynamic-api-individuals-solr-test.n3

```diff
@@ -0,0 +1,59 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#> .
+@prefix dynapi: <https://vivoweb.org/ontology/vitro-dynamic-api#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/solrQuery/genericSolrTextQuery>
+        a                        dynapi:operation, dynapi:solrQuery ;
+        rdfs:label               "Test solr query 1"@en-US ;
+        dynapi:requiresParameter <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/textSearchParam> ;
+        dynapi:solrQueryText   "http" ;
+        dynapi:solrField   "Field1" ;
+        dynapi:solrField   "Field2" ;
```

Comment:
Usually this is structured as:
dynapi:solrField    "Field1" , "Field2" ;

or

dynapi:solrField    "Field1" , 
                             "Field2" ;